framework/PageFramework.css

```diff
@@ -1,7 +1,5 @@
 #app {
-  position: fixed;
-  width: 100%;
-  height: 100%;
+  min-block-size: 100dvh;
```

Comment:
i think 100dvh breaks mobile. You have to use 100%. Needs testing.

Reply:
I've reverted this change for now... it's unrelated to the forms stuff. I'll take a closer look at this in a separate PR